### PR TITLE
Allow running as unprivileged user

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/7349

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,12 +3,21 @@ FROM python:3.7-alpine3.9
...
 
 ADD requirements.txt /
+RUN pip install --upgrade pip
 RUN pip install -r /requirements.txt
 
 WORKDIR /app
 COPY indexer.py /app/
 COPY mapping.json /app/
+
+USER indexer
```
Comment (paurosello) [inline Dockerfile]: In order to be able to use this image in Kubernetes without the SecurityContext we should put `USER 101`

Comment (paurosello) [inline Dockerfile]:
```suggestion
USER 101
```

Comment (marians) [reply Dockerfile]: Why? It works both ways

Comment (marians) [reply Dockerfile]: oh

Comment (paurosello) [reply Dockerfile]: A "security" feature (a nominal user could still have uid 0 and be root)

https://kubernetes.io/docs/concepts/policy/pod-security-policy/#users-and-groups

**Comment by marians** [comment]
Are we fine here?

**Pull request merged**